packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -139,6 +147,9 @@ const DropdownMenuTrigger = React.forwardRef((props, forwardedRef) => {
         // only call handler if it's the left button (mousedown gets triggered by all mouse buttons)
         // but not when the control key is pressed (avoiding MacOS right click)
         if (event.button === 0 && event.ctrlKey === false) {
+          // prevent trigger focusing when opening
+          // this allows the content to be given focus without competition
+          if (!context.open) event.preventDefault();
```

Comment:
In non modal mode the removal of trapping seems to let the trigger steal focus from content on open

Reply:
I'm confused about this change. It feels like there's a deeper issue here if something is happening.

Reply:
It makes sense to me I think 💭 

The `mousedown` event waits for your handler to execute so it can check if you have called `event.preventDefault()` to prevent focusing. Therefore, when you mousedown on the trigger button: 

- it opens the menu 
- `FocusScope` kicks in and opens the container
- the browser `mousedown` handler checks if `event.preventDefault` was called
- it wasn't, so focuses the trigger

I do wonder if this would still be a problem if we used `onPointerDown` though instead. Did you try that already @andy-hook since we want to move in that direction anyway?

Reply:
I'm seeing exactly what @jjenzz is describing for either events, though it's a good point that this should probably be a pointer event, the underlying issue remains though.

Reply:
I'm just confused why this issue is new and wasn't present before I guess.

Reply:
Yeah, although I would have thought there was something covering it up before (a race condition perhaps) and that this is more accurate given we know that `onMouseDown` will focus the button.

Reply:
afaik these were always focus trapped before, now without a trap the focus is allowed to shift back to the trigger

Reply:
https://codesandbox.io/s/eloquent-williamson-qwp9y seems it should be working, at least in the current configuration 🤔 

That said, if we switch to `pointerDown` we will have the same issue. But I am curious what's happening under the hood here.